### PR TITLE
UND-70-Tab-Pagination

### PR DESCRIPTION
**Pull request by harishkumar-srijan**
Tab Pagination

### CODE REVIEW

File: stories/Molecules/Navbar/Pagination/pagination.scss [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+/* Tab start */
+
+.pagination__wrapper {
+  align-items: center;
+  display: flex;
+  list-style: none;
+  padding-left: 0;
+
+  .page__item {
```
Comment (surbhi90) [inline stories/Molecules/Navbar/Pagination/pagination.scss:12]: @harishkumar-srijan We can keep the page__item outside of pagination__wrapper class

File: stories/Molecules/Navbar/Pagination/Pagination.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./pagination.scss";
+
+export const Pagination = ({ text, text2 }) => {
+  return (
+    <ul className="pagination__wrapper">
```
Comment (surbhi90) [inline stories/Molecules/Navbar/Pagination/Pagination.jsx:6]: @harishkumar-srijan it should be pagination-wrapper class it is not element on any block

File: stories/Molecules/Navbar/Tab/Tab.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+export const Tab = ({ tab1, tab2, tab3, data, data2, data3 }) => {
+  useEffect(() => {
+    TabFun();
+  }, []);
+
+  return (
+    <div >
+      <ul className="tab__items" id="tab">
+        <li className="tab__item active" data-tab="tab-1"> { tab1 } </li>
```
Comment (surbhi90) [inline stories/Molecules/Navbar/Tab/Tab.jsx:13]: @harishkumar-srijan it should have map loop

File: stories/Molecules/Navbar/Tab/Tab.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        <li className="tab__item" data-tab="tab-2"> { tab2 } </li>
+        <li className="tab__item" data-tab="tab-3"> { tab3 } </li>
+      </ul>
+      <div id="tab-1" className="tab__content active"> { data } </div>
```
Comment (surbhi90) [inline stories/Molecules/Navbar/Tab/Tab.jsx:17]: @harishkumar-srijan it should also have map function

File: stories/assets/js/Tab_custom.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+/* Tab JS start custom */
+export function TabFun() {
+	$('#tab.tab__items .tab__item').click(function(){
```
Comment (surbhi90) [inline stories/assets/js/Tab_custom.js:3]: @harishkumar-srijan i think we can give '#tab .tab__item'

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

File: stories/assets/js/Tab_custom.js (hunk 5 above)
Comment (harishkumar-srijan) [reply stories/assets/js/Tab_custom.js]: Done

**Review by surbhi90 — APPROVED** [review]

**Pull request merged**